arrows/vxl/high_pass_filter.cxx

```diff
@@ -0,0 +1,564 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "high_pass_filter.h"
+
+#include <arrows/vxl/image_container.h>
+#include <vital/util/enum_converter.h>
+
+#include <vil/vil_convert.h>
+#include <vil/vil_image_view.h>
+#include <vil/vil_math.h>
+#include <vil/vil_plane.h>
+#include <vil/vil_transpose.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace vxl {
+
+namespace { // anonymous
+
+enum filter_mode
+{
+  MODE_box,
+  MODE_bidir,
+};
+
+ENUM_CONVERTER( mode_converter, filter_mode,
+                { "box", MODE_box }, { "bidir", MODE_bidir } )
+
+// ----------------------------------------------------------------------------
+template <typename T> struct accumulator;
+
+template <> struct accumulator<char>
```

Comment:
Pedantic, but my understanding is the whitespace between opening and closing <>.

Also, I find particular line is a bit confusing. My understanding is char can be signed or unsigned depending on implementation but int is always signed. So maybe if my understanding is correct, being consistent across this and the following line would be ever so slightly clearer.